cloudferry_devlab/cloudferry_devlab/tests/config.py

```diff
@@ -586,6 +586,8 @@
     {'name': 'server4', 'image': 'deleted_image', 'flavor': 'flavorname2'},
     {'name': 'server5', 'image': 'image1', 'flavor': 'flavorname1'},
     {'name': 'not_in_filter', 'image': 'image1', 'flavor': 'flavorname1'},
+    {'name': 'with_config_drive', 'image': 'image1', 'flavor': 'flavorname1',
+     'config_drive': True},
```

Comment:
I think you can add this parameter to another VM which described above or below, increasing amount of VMs is not necessary for this case and also increasing load on devlab and jobs's execution time.